rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/report/SW360ReportService.java

```diff
@@ -277,15 +269,27 @@ public ByteBuffer getLicenseInfoBuffer(User sw360User, String id, String generat
         return licenseInfoFile.bufferForGeneratedOutput();
     }
 
+    private List<String> getSelectedReleaseRationships(String selectedRelRelationship) {
+        List<String> selectedReleaseRelationships = null;
+        if (!CommonUtils.isNullEmptyOrWhitespace(selectedRelRelationship)) {
+            selectedReleaseRelationships = Arrays.asList(selectedRelRelationship.split(","));
+        }
+        return selectedReleaseRelationships;
```

Comment:
Would be nice to add a check of relationship values against allowed values here?